aviary/validation_cases/benchmark_tests/test_bench_FwGm.py

```diff
@@ -14,19 +14,18 @@
 class ProblemPhaseTestCase(unittest.TestCase):
     """
     Test the setup and run of a large single aisle commercial transport aircraft using
-    FLOPS mass method and TWO_DEGREES_OF_FREEDOM mission method. Expected outputs
-    based on 'models/test_aircraft/aircraft_for_bench_FwFm.csv' model.
+    FLOPS mass method, GASP aero method, and TWO_DEGREES_OF_FREEDOM mission method.
+    Expected outputs based on 'models/test_aircraft/aircraft_for_bench_FwFm.csv' model.
     """
 
     def setUp(self):
         _clear_problem_names()  # need to reset these to simulate separate runs
 
     @require_pyoptsparse(optimizer='IPOPT')
     def bench_test_swap_3_FwGm_IPOPT(self):
-        local_phase_info = deepcopy(phase_info)
         prob = run_aviary(
             'models/test_aircraft/aircraft_for_bench_FwGm.csv',
-            local_phase_info,
+            phase_info,
```

Comment:
I think we want to leave the deepcopy here because Aviary still might be making changes to the phase-info.